{"CVE Code": "CVE-2017-5601", "Description": "An error in the lha_read_file_header_1() function (archive_read_support_format_lha.c) in libarchive 3.2.2 allows remote attackers to trigger an out-of-bounds read memory access and subsequently cause a crash via a specially crafted archive.", "NVD Link": "https://nvd.nist.gov/vuln/detail/CVE-2017-5601", "Vulnerability Categories": "N/A", "Affected Products": "\nLibarchive\u00bbLibarchive\u00bbVersion:3.2.2cpe:2.3:a:libarchive:libarchive:3.2.2:*:*:*:*:*:*:*Matching versions", "EPSS Score": "0.63% Probability of exploitation activity in the next 30 days\n~ 76 % Percentile, the proportion of vulnerabilities that are scored at or less", "EPSS History": "\n #  Date  Old EPSS Score  New EPSS Score  Delta (New - Old) \n 1  2023-03-07  1.21%  0.63%  -0.58 ", "CVSS Scores": "5.0", "CWE": [{"CWE_ID": "CWE-125 Out-of-bounds Read", "CWE_link": "/cwe-details/125/Out-of-bounds-Read.html"}], "Reference": [{"ref_link": "https://lists.debian.org/debian-lts-announce/2018/11/msg00037.html", "ref_desc": "[SECURITY] [DLA 1600-1] libarchive security updateCVEs referencing this url", "ref_cve_link": "https://www.cvedetails.com/reference-url-info/coB5WdFCBupvcJ1IrhQ5PqwLAQo.html"}, {"ref_link": "https://github.com/libarchive/libarchive/commit/98dcbbf0bf4854bf987557e55e55fff7abbf3ea9", "ref_desc": "Fail with negative lha->compsize in lha_read_file_header_1() \u00b7 libarchive/libarchive@98dcbbf \u00b7 GitHubIssue Tracking;Patch;Third Party Advisory", "ref_cve_link": "N/A"}, {"ref_link": "http://www.securitytracker.com/id/1037974", "ref_desc": "libarchive Out-of-Bounds Memory Read Error in lha_read_file_header_1() Lets Remote Users Cause the Target Service to Crash - SecurityTracker", "ref_cve_link": "N/A"}, {"ref_link": "http://www.securityfocus.com/bid/95837", "ref_desc": "libarchive 'lha_read_file_header_1()' Function Memory Corruption VulnerabilityThird Party Advisory;VDB Entry", "ref_cve_link": "N/A"}]}